src/pages/WorkspaceSwitcherPage.tsx

```diff
@@ -218,6 +218,7 @@ function WorkspaceSwitcherPage({policies}: WorkspaceSwitcherPageProps) {
                             accessible={false}
                             role={CONST.ROLE.BUTTON}
                             onPress={() => {
+                                Navigation.goBack();
```

Comment:
Adding the goBack function to the + press event caused an issue https://github.com/Expensify/App/issues/46383 where, on Android and mobile web devices, the LHN briefly appears before navigating to the workspaces page.